docs/eventing/getting-started.md

```diff
@@ -0,0 +1,438 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative workflow. You will learn how to create, send, and verify events in Knative.
```

Comment:
Great job on adding more explanation! I think you addressed a lot of Richie's concerns.